server/service/src/invoice_line/stock_out_line/update/validate.rs

```diff
@@ -38,8 +38,8 @@ pub fn validate(
         input.stock_line_id.clone(),
         connection,
     )?;
-    if unique_stock.is_some() {
-        return Err(StockLineAlreadyExistsInInvoice(unique_stock.unwrap().id));
+    if let Some(unique_stock) = unique_stock {
```

Comment:
will have to go through and refactor the function that returns this since misleading...

Reply:
What do you mean? maybe just rename `unique_stock` to `existing_stock_line`?

Reply:
Yes and the function is called (check_unique_stock_line)